language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ExtractToLocalVarCodeAction.java

```diff
@@ -59,7 +61,7 @@ public class ExtractToLocalVarCodeAction implements RangeBasedCodeActionProvider
 
     public static final String NAME = "Extract To Local Variable";
     private static final String VARIABLE_NAME_PREFIX = "var";
-
+    
```

Comment:
Is this extra tab intentional? is it from the checkstyles?

Reply:
That's probably when we apply `ctrl + alt +L` to format the entire file. @LakshanWeerasinghe you are using the WSO2 coding styles right?

Reply:
@IMS94 yes I am

Reply:
@IMS94 yes, I have used that. but haven't come across indentations after constants though. Are there any general style practices like this?

Reply:
Although I haven't select `keep indents on empty lines`, intellij indents empty lines. Do I need to make this change? @IMS94